crypto/threads_pthread.c

```diff
@@ -7,24 +7,379 @@
  * https://www.openssl.org/source/license.html
  */
 
+#include <signal.h>
+#include <unistd.h>
```

Comment:
unistd.h isn't universal.  MSVC complains that it's missing.  `#include "e_os.h"` does a better job of selecting when to include this and when not.

Reply:
Changed, thanks!